aiida/cmdline/commands/cmd_node.py

```diff
@@ -61,6 +64,86 @@ def repo_ls(node, relative_path, color):
         echo.echo_critical(exception)
 
 
+@verdi_node_repo.command('dump')
+@click.option(
+    '-f', '--force', help='Replace existing directories and files without prompting for confirmation.', flag_value=True
+)
+@arguments.NODE()
+@click.argument(
+    'output_directory',
+    type=click.Path(exists=True, file_okay=False, writable=True),
+    required=True,
+)
+@with_dbenv()
+def repo_dump(node, output_directory, force):
+    """Copy the repository files of a node to an output directory."""
+    from aiida.orm.utils.repository import FileType
+
+    output_directory = pathlib.Path(output_directory)
+
+    output_directory.mkdir(exist_ok=True)
```

Comment:
so, does it need to exist or not? In the option I see `exists=True`
I'm not sure we want to be creating new base directories...

Reply:
removed the ``mkdir``, indeed this is checked by click.